### PR TITLE
Document CLI options in README.

### PR DESCRIPTION
**Pull request by cjcenizal**

### CODE REVIEW

**Comment by cjcenizal** [comment]
@sqren Mind taking a look?

File: readme.MD [hunk 1]
```diff
@@ -10,16 +10,29 @@ A simple CLI tool for backporting commits
...
 
+### Options
+
+| Option | Description | Accepts |
+| --- | --- | --- |
+| --multiple     | Backport to multiple versions |         [boolean] [default: true] |
+| --own          | Only show own commits |                 [boolean] [default: true] |
+| --config       | Show configuration file |                               [boolean] |
+| --help         | Show help |                                             [boolean] |
+| -v, --version  | Show version number |                                   [boolean] |
```
Comment (sqren) [inline readme.MD]: Awesome! I don't think we need to write `[boolean]` for `config`, `help` and `version`. Apart from that 👍

**Comment by cjcenizal** [comment]
Haha good point. Updated:

image

**Comment by sqren** [comment]
@cjcenizal Sweet, thanks!

**Pull request merged**